src/adapters/communities-db.ts

```diff
@@ -223,6 +223,17 @@ export function createCommunitiesDBComponent(
       return pg.exists(query, 'exists')
     },
 
+    async getCommunitiesByPlaceId(placeId: string): Promise<string[]> {
```

Comment:
Are we aware of the limitation that a Place can only be linked to a single Community? I checked the migration because of the `id` reference here and we are using it as PK